### PR TITLE
feat: improved tag order

### LINKED ISSUE #49: Tag order: Tags between `@param` and `@returns`
Right now, there can be [a lot of stuff](https://github.com/hosseinmd/prettier-plugin-jsdoc/blob/d34af57c59ccee880862f8fc36582f540e71acc6/src/roles.ts#L151) between `@param` and `@return` tags. Why?

### PR DESCRIPTION
**Pull request by RunDevelopment**
This fixes #49.

### CODE REVIEW

File: tests/__snapshots__/template.test.js.snap [hunk 1]
```diff
@@ -6,9 +6,9 @@ exports[`extends 1`] = `
  *
```
Comment (RunDevelopment) [inline tests/__snapshots__/template.test.js.snap:6]: @hosseinmd I just noticed this. It's this formatting of `@class` and the text below a bug?

Comment (hosseinmd) [reply tests/__snapshots__/template.test.js.snap:6]: Yes this is a bug, but I can't fix. And i don't know why

Comment (RunDevelopment) [reply tests/__snapshots__/template.test.js.snap:6]: Ok. Just wanted to ask. I'll open an issue for it and try to fix it in the next few days.

**Review by hosseinmd — APPROVED** [review]

**Comment by hosseinmd** [comment]
Is this a breaking change? I'm asking because of versioning

**Comment by RunDevelopment** [comment]
Depends on what you define as "breaking".

**Comment by RunDevelopment** [comment]
If that was the case, then any change we make to our formatting is breaking, even dfc78b40edd088d0c03afa8d76b6d61feb0f8383. So is dfc78b40edd088d0c03afa8d76b6d61feb0f8383 a breaking change?

[Semantic versioning](https://semver.org/) requires a new major version for "incompatible API changes". Do we have an API?

**Comment by hosseinmd** [comment]
Ok, so please resolve conflict.

**Pull request merged**